Automattic/vip-go-mu-plugins
# Add Object Cache panel to Query Monitor

## Description

Adds panel to Query Monitor that shows Object Cache stats. This the same data as currently shown in Debug Bar.

## Changelog Description

Show Object Cache stats to Query Monitor

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

Install and view:

![qm-object-cache](https://user-images.githubusercontent.com/251183/168381110-a5c406a7-fb7d-44aa-ad47-21c61d3d41e4.png)

Enable Debug Bar and notice "Memcache" item not shown while QM is active:

![debug-bar-no-cache](https://user-images.githubusercontent.com/251183/168381112-1196c555-1701-4e8f-a409-deae80a79537.png)
